pkg/kancontroller/kancontroller.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2019 The Kanister Authors.
```

Comment:
```suggestion
Copyright 2020 The Kanister Authors.
```
Should it be 2020 for new files?